api/powershell/invite_members_to_org.ps1

```diff
@@ -0,0 +1,69 @@
+#Requires -version 7
+
+<#
+.SYNOPSIS
+Batch invite members to an organization
+
+.DESCRIPTION
+This script runs a batch organization invitation process for a given list of GitHub enterprise
+consumed licenses.
+
+The input is a CSV file with a column named "Handle or email", such as can be exported from the
+Enterprise settings > Enterprise licensing page. Users with appropriate permissions can export
+the CSV file, edit it in their favorite spreadsheet to select emails to invite, then use this
+script to invite them to an org.
+
+.PARAMETER LicensesFile
+The path of the consumed licenses CSV.
+
+.PARAMETER Organization
+The name of the organization to invite members to.
+
+.PARAMETER PAT
+The personal access token. It must have "admin:org" scope to be authorized for the operation.
+
+.EXAMPLE
+.\invite_members_to_org.ps1 -LicensesFile .\consumed_licenses.csv -Organization my-organization -PAT xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+#>
+
+param (
+  [string] [Parameter(Mandatory=$true)] $LicensesFile,
+  [string] [Parameter(Mandatory=$true)] $Organization,
+  [string] [Parameter(Mandatory=$true)] $PAT
+)
+
+Import-Csv $LicensesFile | ForEach-Object {
+    Write-Host "---------------------------------------------"
+
+    $Body = @{}
+    if ($_."Handle or email" -Match "@") {
+        Write-Host "Inviting email $($_."Handle or email")..."
+        $Body.email = $_."Handle or email"
+    } else {
+        Write-Host "Inviting handle $($_."Handle or email")..."
+        $HandleIdRequest = Invoke-RestMethod -SkipHttpErrorCheck -Uri "https://api.github.com/users/$($_."Handle or email")"
```

Comment:
Do we need that check here? Wouldn't the invitations call later on fail if we have an invalid handle?

Reply:
Good point, the check is indeed not needed. I added it to facilitate some debugging, but ended up leaving it, since it adds no harm.

Reply:
I feel it makes the script a bit more complicated than necessary but I am fine either way.